### ISSUE TITLE
Cannot use "pwd" and "ssh" command. "ls" works fine though.

### ISSUE THREAD

**Issue by ChendiDotLin** [IBM/kui#1124]
![519111554950377_ pic](https://user-images.githubusercontent.com/21987873/55927288-872b2900-5be2-11e9-9675-82a1fc16859c.jpg)

**Comment by ChendiDotLin** [comment]
Downloaded the "double clickable mac os" version

**Comment by starpit** [comment]
thanks for the bug report! we will check this out first thing in our morning. can you give us the output of `version`? thanks.

**Comment by ChendiDotLin** [comment]
2.25.0-dev.1568.20190411003125.

Thanks for your reply.

**Comment by starpit** [comment]
great, thanks for the info. we'll get this squared away asap.

**Comment by starpit** [comment]
this one is currently blocked on https://github.com/daviwil/node-pty-prebuilt/issues/16

we will see if we can work around it if that issue is not resolved very soon

**Comment by wanghh2000** [comment]
Make it run in browser and cannot execute command too. Is it the same root cause?

**Comment by starpit** [comment]
hi @wanghh2000 thanks for this update.

there are two separate issues here. the original issue as reported affects local electron builds. this is due to one of our npm dependencies not having an up-to-date build for the version of electron that we use.

there is a separate issue that affects raw terminal operations (such as `pwd`) when run in a browser. firstly, to use these, or `kubectl` commands, in a browser, yiou will need to have a kui [proxy](https://github.com/IBM/kui/tree/master/packages/proxy), combined with this PR #1171 that is still (as of this writing) in flight

**Comment by starpit** [comment]
@ChendiDotLin as of build 1643, electron builds for mac should now support the `pwd` and other shell-oriented commands.

related PR #1178
